src/regressors/linear_regression.cpp

```diff
@@ -3,6 +3,10 @@
 
 namespace cpp_ml::models::regressors {
 
+linear_regression :: linear_regression ()
+	:degree_(1)
+{}
+
 linear_regression :: ~linear_regression() {}
 
 auto linear_regression :: predict(const std::vector<double> & instance) const -> double {
```

Comment:
The three methods from this line to the end of the namespace should be deleted (it will automatically call the parent implementation)